core/io/file_access_pack.cpp

```diff
@@ -88,7 +88,11 @@ void PackedData::add_path(const String &pkg_path, const String &path, uint64_t o
 				}
 			}
 		}
-		cd->files.insert(path.get_file());
+		String filename = path.get_file();
+		// Don't add as a file if the path points to a directoryy
```

Comment:
typoo in directoryy

Reply:
Oops. Well, it's already merged. I guess it's not as serious as to revert it. Some day someone will touch this file again and have the chance of fixing it.

Reply:
> I guess it's not as serious as to revert it.

Heh, I had seen it but chose to ignore it, didn't want to wait for a commit amend :)
`codespell` will fix it eventually.